shark-browser.js

```diff
@@ -22,6 +22,7 @@ Shark.AppClient = require('./src/clients/app-client')
 Shark.AssetClient = require('./src/clients/asset-client')
 Shark.ConsentClient = require('./src/clients/consent-client')
 Shark.ConsentChangesClient = require('./src/clients/consent-changes-client')
+Shark.ContactClient = require('./src/clients/contact-client')
```

Comment:
do we list all clients somewhere in README? If yes, please add it.

Reply:
only in changelog